pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "sample-size"
-version = "0.0.1a1"
+version = "1.0.0"
```

Comment:
It is the time we want people to start using it so I think 1.0.0 makes sense https://semver.org/#how-do-i-know-when-to-release-100